test/kino/data_table_test.exs

```diff
@@ -275,6 +275,21 @@ defmodule Kino.DataTableTest do
     })
   end
 
+  test "supports a formatter option" do
+    entries = %{x: 1..3, y: [1.1, 1.2, 1.3]}
+    formatter = &Kino.DataTable.Formatter.format/2
```

Comment:
Let's keep it simple for the test and define a formatter inline:

```suggestion
    formatter =
      fn
        {:__header__, value} -> "h:#{value}"
        {:x, value} -> "x:#{value}"
        _ -> :default
      end
```

Reply:
Done!